src/main/java/rs/teslaris/core/harvester/product/Product.java

```diff
@@ -0,0 +1,56 @@
+package rs.teslaris.core.harvester.product;
+
+import java.util.List;
+import javax.xml.bind.annotation.XmlAccessType;
+import javax.xml.bind.annotation.XmlAccessorType;
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlElementWrapper;
+import javax.xml.bind.annotation.XmlRootElement;
+import javax.xml.bind.annotation.XmlType;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import rs.teslaris.core.harvester.common.MultilingualContent;
+import rs.teslaris.core.harvester.common.PersonAttributes;
+
+@XmlType(name = "TProduct", namespace = "https://www.openaire.eu/cerif-profile/1.1/")
+@XmlAccessorType(XmlAccessType.FIELD)
+@XmlRootElement(name = "Product")
+@Getter
+@Setter
+@NoArgsConstructor
+@AllArgsConstructor
+@ToString
+public class Product {
+
+    @XmlElement(name = "Keyword")
+    List<String> keywords;
+
+    @XmlAttribute(name = "id")
+    private String id;
+
+    @XmlElement(name = "Type", namespace = "https://www.openaire.eu/cerif-profile/vocab/COAR_Product_Types")
+    private String type;
+
+    @XmlElement(name = "Language")
+    private String language;
+
+    @XmlElement(name = "Name")
+    private MultilingualContent name;
+
+    @XmlElement(name = "URL")
+    private MultilingualContent url;
```

Comment:
This should be a List, but not sure it is a List<MultilingualContent>, it might be just list of String

Reply:
I think URLs had a language attribute to show in which language the webpage is, we do not have that kind of system in our CRIS but we can parse that information in order to have it in case we decide to encorporate that into our system.

Reply:
I think we need to preserve language agnostic URL in our database (which will mean we should define source domain and target domain and do replacement in URLs). Of course later we should implement setting locale (language) in the URL to display the record page. Something similar as this [PR in VIVO](https://github.com/vivo-project/Vitro/pull/419) :-)